packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -58,14 +58,16 @@ const StyledLink = styled.a`
   }
 
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    ${({ script }) => script && getGreatPrimer(script)};
+    ${({ script, typography }) =>
+      script && typography !== 'pica' && getGreatPrimer(script)}
   }
 `;
 
-const ALIGN_RANK_TO_LINK_SPACING = '0.375rem';
+const ALIGN_RANK_TO_LINK_SPACING = rankTypography =>
+  rankTypography === 'trafalgar' ? '0.1rem' : '0.375rem';
 
 const StyledItem = styled.div`
-  padding-top: ${ALIGN_RANK_TO_LINK_SPACING};
+  padding-top: ${props => ALIGN_RANK_TO_LINK_SPACING(props.rankTypography)};
```

Comment:
We could improve the readability of this function slightly with a sprinkle of de-structuring, like you've used in your other functions:
```
padding-top: ${({ rankTypography }) => ALIGN_RANK_TO_LINK_SPACING(rankTypography)};
```